docs/browse-list.md

```diff
@@ -18,6 +18,18 @@ For catalog production, ssh to catalog-indexer1. Confirm through the crontab tha
 
 Expected time: 5.5 - 6 hours.
 
+To regenerate the **subject lists** used in `Subject (browse)` in Orangelight, SSH as `deploy` to the machine used to produce the browse lists (`catalog-indexer1|2|3`) and run:
+
+```
+cd /opt/orangelight/current
+OL_DB_PORT=5432 bundle exec rake browse:subjects
+OL_DB_PORT=5432 bundle exec rake browse:load_subjects
+```
```

Comment:
if you mean to run them separately then specify that the first one needs to finish and generate the csv file; about 3hours. The second one doesn't take much time because it loads the csv to postgres.

or add the commands together with &&